service/src/main/java/uk/gov/hmcts/reform/fpl/NotifyTemplates.java

```diff
@@ -18,7 +18,7 @@ private NotifyTemplates() {
     public static final String CMO_ORDER_ISSUED_DOCUMENT_LINK_NOTIFICATION_TEMPLATE =
         "931dcd86-d331-4273-abaa-208daed0761a";
     public static final String CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE =
-        "6bdbc939-14bb-4dae-ac35-e0d381840719";
+        "58aa7e13-e80a-4630-a610-ee7fc1b6f9f9";
```

Comment:
Why did we need an entirely new template? Or does notify generate a new ID with edits?

Reply:
I think we are going with this process of not changing live templates. spoke to @joez-f about this one. This way we ensure we don't break anything. When QA is happy with this story I will remove the old template. happy with this?

Reply:
Ah ok yeah that makes sense. Happy with that 👍